highfidelity/hifi
# Case 22069: Update Controller Help Screen

In light of the recent changes to locomotion controls, this is a temporary update to the controller help screen that links the users to our controller documentation.

[Relevant MS ticket.](https://highfidelity.fogbugz.com/f/cases/22069)

**Test Plan:**
_Part 1:_
- Launch interface in desktop mode.
- Go to _Help -> Controls Reference_
- New graphic should display.
- Click the image of the keyboard, a new window should launch outside interface that goes to this URL:
https://docs.highfidelity.com/en/rc81/explore/get-started/desktop.html#movement-controls

_Part 2:_
- Re-launch interface with the vive
- Go to _Help -> Controls Reference_
- Graphic for the Vive should display.
- Click the image of the controllers and a new window should launch outside interface that goes to this URL:
https://docs.highfidelity.com/en/rc81/explore/get-started/vr-controls.html#vr-controls

_Part 3:_
- Repeat steps for Part 2 with the Oculus Touch. The Oculus graphic should display, the URL will be the same as Part 2.

_Part 4:_
- Repeat the steps for Part 2 with the WindowsMR. The WindowsMR graphic should display, the URL will be the same as Part 2.
